diktat-rules/src/test/kotlin/org/cqfn/diktat/util/FixTestBase.kt

```diff
@@ -58,6 +68,85 @@ open class FixTestBase(protected val resourceFilePath: String,
         )
     }
 
+    private fun getSaveForCurrentOs() = when {
+        System.getProperty("os.name").startsWith("Linux", ignoreCase = true) -> "save-$SAVE_VERSION-linuxX64.kexe"
+        System.getProperty("os.name").startsWith("Mac", ignoreCase = true) -> "save-$SAVE_VERSION-macosX64.kexe"
+        System.getProperty("os.name").startsWith("Windows", ignoreCase = true) -> "save-$SAVE_VERSION-mingwX64.exe"
+        else -> ""
+    }
+
+    private fun downloadFile(url: String, file: File) {
+        val httpClient = HttpClients.createDefault()
+        val request = HttpGet(url)
+        val response: CloseableHttpResponse = httpClient.execute(request)
+        val fileSave = response.entity
+        fileSave?.let {
+            FileOutputStream(file).use { outstream -> fileSave.writeTo(outstream) }
+        }
+        response.close()
+        httpClient.close()
```

Comment:
It's a classic situation when you'll get opened resources if you get an exception before. You should wrap usages of resources in `use`